keps/sig-node/4383-dra-semantic-model-mode-switching/kep.yaml

```diff
@@ -0,0 +1,46 @@
+title: KEP Template
+kep-number: 4383
+authors:
+  - "@pohly"
+owning-sig: sig-node
+participating-sigs:
+  - sig-scheduling
+  - sig-autoscaling
+status: implementable
+creation-date: 2024-01-05
+reviewers:
+  - TBD
+  - "@alice.doe"
+approvers:
+  - TBD
+  - "@oscar.doe"
+
+see-also:
+  - "/keps/sig-node/3063-dynamic-resource-allocation"
+  - "/keps/sig-node/4381-dra-semantic-parameters"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.30"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.30"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: DRACounterSemanticModel
+    components:
+      - kube-scheduler
+      # kube-apiserver, kubelet and kube-controller-manager are agnostic to
+      # which numeric model is being used.
```

Comment:
Which might not be true, depending on https://github.com/kubernetes/enhancements/pull/4384#discussion_r1472071508. We need to conclude that discussion and then update here.